data/isdv4-5148.tablet

```diff
@@ -0,0 +1,16 @@
+# this is for the Wacom pen + touchscreen as found in the Lenovo ThinkPad X1 Yoga (3rd-gen)
```

Comment:
can you add a link to https://github.com/linuxwacom/wacom-hid-descriptors/issues/117 here please, so we can find this easier in the future. See the `data/elan-2514.tablet` file for an example.